src/controllers/company.details.controller.ts

```diff
@@ -47,6 +47,13 @@ export const confirmCompanyStartRequest = async (req: Request, res: Response, ne
     const token: string = req.chSession.accessToken() as string;
     if (token) {
       const company: ExtensionsCompanyProfile = await getCompanyProfile(companyNumber, token);
+
+      const isFilingDateEligible: boolean = isExtensionDueDateWithinLimit(company);
```

Comment:
you can remove the boolean and just cal the function in the if

Reply:
removed line